custom_modules/application/application.js

```diff
@@ -45,7 +46,11 @@ class ApplicationEngine {
     this.window.showLoader();
     const res = this.foodAPIConnection.getItemByID(id);
     res.then((data) => {
-      this.window.displayItem(data.meals[0]);
+      const iRes = this.involvmentConnection.getComments(id);
+      iRes.then(() => {
+        this.currSelectedItemID = id;
+        this.window.displayItem(data.meals[0]);
```

Comment:
- [ ] Could comments be loaded initially after user selects a particular item? what happens for example if we: 
        `iRes.then((comments) => {
          this.window.displayItemComments(comments);
        });`
       so as to make sure the comments loads with everything else and not rely on mandatory POST of new comment to get the list.